wasmtime-jni/src/io_github_kawamuray_wasmtime_Memory/imp.rs

```diff
@@ -47,4 +47,10 @@ impl<'a> JniMemory<'a> for JniMemoryImpl {
         let mem = interop::get_inner::<Memory>(&env, this)?;
         Ok(mem.size() as jint)
     }
+
+    fn grow(env: &JNIEnv, this: JObject, page: jint) -> Result<jint, Self::Error> {
+        let mem = interop::get_inner::<Memory>(&env, this)?;
+        let re = mem.grow(page as u32)? as i32;
```

Comment:
1. `as jint` instead.
2. Just `Ok(mem.grow(page as u32)? as jint)` should be fine, or please give it descriptive name (like `prev_pages`) if you intend to use this temporary variable for that purpose.